docs/xxhandlesigusr2.md

```diff
@@ -0,0 +1,57 @@
+<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]HandleSIGUSR2
+
+**(AIX&reg;, Linux&reg;, macOS&reg;, and z/OS&reg; only)**
+
+ This option affects the handling of the operating system signal `SIGUSR2`. This signal is user-defined and triggers the [`user2`](xdump.md#dump-events) event, which is commonly used for taking system dump files with exclusive access. The operating system installs a default signal handler for handling `SIGUSR2`. If there is no VM or application signal handler, then the operating system's signal handler is used.
+
+
+## Syntax
+
+        -XX:[+|-]HandleSIGAUSR2
+
+| Setting               | Effect  | Default                                                                            |
+|-----------------------|---------|:----------------------------------------------------------------------------------:|
+| `-XX:+HandleSIGUSR2` | Enable  | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+| `-XX:-HandleSIGUSR2` | Disable |                                                                                    |
+
+
+## Explanation
+
+When enabled, the VM handles the signal `SIGAUSR2` and generates the various dump files.
```

Comment:
By default SIGUSR2 doesn't do anything. This could say: and generates the user2 event, which can be configured with -Xdump to invoke a dump agent.

Reply:
Done